spec/views/results_page/client_financial_content_spec.rb

```diff
@@ -204,16 +204,8 @@
       end
     end
 
-    context "when dependants allowance is not positive figure" do
-      let(:dependant_allowance) { 0 }
-
-      it "does not display the dependants allowance field" do
-        expect(page_text).not_to include "Dependants allowance"
-      end
-    end
-
-    context "when dependants allowance is nil" do
-      let(:dependant_allowance) { nil }
+    context "when there are no dependants" do
```

Comment:
question - do we want to include `let(:child_dependants) { false }` so we can say that there were no dependants at all, or is that overthinking it?

Reply:
It's a fair point. We might as well explicitly set child_dependants to false in the session data, so that way it's very clear that there are no dependants when there are no dependants.